unit/widgets/textbox_spec.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+require 'rspec'
+require_relative '../../../lib/libyui_client/http/response'
+
+
+module LibyuiClient
+  module Widgets
+    RSpec.describe Textbox do
+      describe '.Textbox' do
+        
+        context 'when be initialized' do
+          it 'should be a Textbox instance with default timeout and interval' do
+            widget_controller = double('widget_controller')
+            widget_filter = double()
+            @textbox = Textbox.new(widget_controller, widget_filter)
+            expect(@textbox).to be_instance_of(Textbox)
+            expect(@textbox.instance_eval('@timeout')).to equal(5)
+            expect(@textbox.instance_eval('@interval')).to equal(0.5)
+          end
+        end
+      end
+      describe '#set' do
+        it 'can set value' do
+          widget_controller = double('widget_controller')
+
+          widget_filter = double("Filter", :id => 'aliases')
+          @textbox = Textbox.new(widget_controller, widget_filter)
+          expect(widget_controller).to receive(:send_action).and_return(
+                                                            :body => [{:class => "YInputField",
+                                                              :debug_label => "Host Aliases",
+                                                              :hstretch => true,
+                                                              :id => "aliases",
+                                                              :label => "Hos&t Aliases",
+                                                              :password_mode => false,
+                                                              :value => ""
+                                                                      }])
+          
+          @textbox.set('test')
```

Comment:
i try to accustom myself with the real doubles. it might be a way to validate the new value